src/components/verification-grid/verification-grid.ts

```diff
@@ -1598,6 +1615,54 @@ export class VerificationGridComponent extends WithShoelace(AbstractComponent(Li
     }
   }
 
+  private updateDecisionWhen(subSelection = this.behavingSubSelection): void {
+    // If any of the decision buttons predicate's pass with the current
+    // sub-selection, the button should not be disabled.
+    const decisionElements = this.decisionElements ?? [];
+    for (const decisionElement of decisionElements) {
+      decisionElement.disabled = !subSelection.some((subject) => decisionElement.when(subject));
+    }
+
+    // Each tiles required decisions are determined from the decision buttons
+    // "when" predicates.
+    // Therefore, when we update the decision buttons disabled state, we also
+    // need to update the required decisions.
+    const gridTiles = Array.from(this.gridTiles);
+    for (const tile of gridTiles) {
+      this.updateDecisionWhenForSubject(tile);
+    }
+  }
+
+  private updateDecisionWhenForSubject(tile: VerificationGridTileComponent): void {
+    const subject = tile.model;
+    const decisionElements = this.decisionElements ?? [];
+
+    // Each subject has required decisions that must be completed.
+    // Required decisions are determined from the decision buttons "when"
+    // predicates.
+    // If the predicate doesn't pass for the current decision button, we mark
+    // the task as "not required" in the subject.
+    for (const decisionElement of decisionElements) {
+      const passes = decisionElement.when(subject);
+      if (passes) {
+        subject.setDecisionRequired(decisionElement.decisionConstructor);
+      } else {
+        subject.setDecisionNoRequired(decisionElement.decisionConstructor);
+      }
+    }
+
+    // We call "requestUpdate" on the tile so that the progress meter is updated
+    // with its new required decisions.
+    // I don't like manually calling "requestUpdate" here, but the component is
+    // an internal component that we control, so while this is an indication of
+    // bad state updates, it's not user facing, so I have just done a dirty
+    // requestUpdate here.
+    // TODO: Correctly fix the update triggers in the tile component so that we
+    // don't have to manually request an update.
+    tile.model = subject;
+    tile.requestUpdate();
```

Comment:
two alternate patterns to consider:

- decisions required in a subject could be a signal. then the tile would know when to update if the signal is being monitored
- we could use a middle-man method on the tile (something like `updateSubject` or `updateSubjectDecision` which could trigger an internal update request

Reply:
I think something like `updateSubjet` would work best for now